ArduPlane/Parameters.h

```diff
@@ -234,7 +234,7 @@ class Parameters {
         // Battery monitoring parameters
         //
         k_param_battery = 166,
-        k_param_rssi_pin,               // unused, replaced by rssi_ library parameters - 167
+        k_param_LQR_controller,               
```

Comment:
You can't replace an old parameter entry. This should probably go to ParametersG2.

Reply:
Do I need to add enumeration in ParametersG2?
Can someone give some pointers, how to do it?

Reply:
No, in G2, you write the number directly in the table.

I don't have any guide for adding parameters to G2, but here is a commit doing it: https://github.com/ArduPilot/ardupilot/commit/35d406aeab75c59e612b16d7b7c5af856800c6cf#diff-f757695534c16617c573ef38443c9ac6